python/paddle/devices/cuda/__init__.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from paddle.fluid import core
+
+from .streams import Stream  # noqa: F401
+from .streams import Event  # noqa: F401
+
+__all__ = [
+    'Stream',
+    'Event',
+    'current_stream',
+    'synchronize',
+]
+
+
+def current_stream(device=None):
+    '''
+    Return the current CUDA stream by the device.
+
+    Parameters:
+        device(paddle.CUDAPlace()|int, optional): The device or the ID of the device which want to get stream from. 
+        If device is None, the device is the current device. Default: None.
+    
+    Returns:
+        CUDAStream: the stream o the device.
```

Comment:
CUDAStream: the stream o the device. ->CUDAStream: the stream of the device.

Reply:
Done, thanks.